docs/source/llms/langchain/index.rst

```diff
@@ -550,3 +550,46 @@ string response, which can be compared to the `"ground_truth"` column.
 
 For a complete example of a LangGraph model that works with this evaluation example, see the 
 `MLflow LangGraph blog <https://mlflow.org/blog/langgraph-model-from-code>`_.
+
+How to control whether my input is converted to List[langchain.schema.BaseMessage] in PyFunc predict?
```

Comment:
Is it possible to log a warning or info message when the request with the converted input failed?

Reply:
Added a debug info, because I don't think users know (or care about) the details of this conversion. If we always show the info it could be confusing to users who don't expect such conversion at all.

Reply:
@serena-ruan Is this issue a rare edge case or it could occur frequently?

Reply:
Emm we haven't received such report for a long time, and the issue we try to fix in this PR is kind of edge case. I think more common case now is that the model accepts `{"messages": ...}` input itself and doesn't need conversion at all.